packages/e2e-tests/specs/editor/various/draggable-block.test.js

```diff
@@ -7,15 +7,14 @@ import {
 	deactivatePlugin,
 	activatePlugin,
 	showBlockToolbar,
-	setBrowserViewport,
 	waitForWindowDimensions,
 	clickBlockAppender,
 } from '@wordpress/e2e-test-utils';
 
 describe( 'Draggable block', () => {
 	// Tests don't seem to pass if beforeAll and afterAll are used.
```

Comment:
Reminder to remove this comment before merging :)

Reply:
done!